qa-tests-backend/src/test/groovy/AdmissionControllerNoImageScanTest.groovy

```diff
@@ -154,12 +154,8 @@ OqxYbK0Iro6GzSmOzxkn+N2AKawLyXi84WSwJQBK//psATakCgAQKkNTAA==
             def timer = new Timer(30, 1)
```

Comment:
I'm a bit leery of this change. You could use a `withRetry()` or such to get the a retry without the catch. But it would be over all exceptions. 
On the other hand, this is probably defensive code or an older fabric8 issue or copy-n-paste junk. Search says this error has not occurred in 14 days:  https://search.ci.openshift.org/?search=Caught+NPE+while+deleting+deployment&maxAge=336h&context=1&type=build-log&name=stackrox&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job
So maybe run with it and we'll see how it flakes or not.

Reply:
> So maybe run with it and we'll see how it flakes or not.

Let's do it